integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/webhooks/subscription/SubscriptionServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package no.difi.meldingsutveksling.webhooks.subscription;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.domain.webhooks.Subscription;
+import no.difi.meldingsutveksling.exceptions.SubscriptionNotFoundException;
+import no.difi.meldingsutveksling.webhooks.WebhookPusher;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.orm.ObjectOptimisticLockingFailureException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.orm.ObjectOptimisticLockingFailureException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)